crates/sui-core/src/authority.rs

```diff
@@ -762,7 +762,7 @@ impl AuthorityState {
             ));
         }
 
-        if !certificate.is_system_tx() && self.is_cert_awaiting_sequencing(certificate)? {
+        if self.is_cert_awaiting_sequencing(certificate)? {
```

Comment:
Just want to make sure that calling `execute_certificate()` on change epoch tx should either return an error, or handle it correctly (not ideal). Maybe return an error instead?

Reply:
I think it should in fact handle it correctly, if it ever happens.

Reply:
Sounds good, if the new logic does handle it correctly.